src/test/java/se/sundsvall/messaging/api/StatusAndHistoryResourceTests.java

```diff
@@ -35,10 +36,16 @@ class StatusAndHistoryResourceTests {
 
     @Mock
     private HistoryService mockHistoryService;
+    @Mock
+    private StatisticsService mockStatisticsService;
 
-    @InjectMocks
     private StatusAndHistoryResource statusAndHistoryResource;
 
+    @BeforeEach
+    void setUp() {
+        statusAndHistoryResource = new StatusAndHistoryResource(mockHistoryService, mockStatisticsService);
```

Comment:
Why is this necessary and why was `@InjectMocks` not a good option (instead of this)?